package.json

```diff
@@ -10,11 +10,23 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@fontsource/roboto": "^5.0.12",
     "@fortawesome/fontawesome-free": "^6.5.2",
+    "@mui/icons-material": "^5.15.15",
+    "@mui/material": "^5.15.15",
+    "@mui/styled-engine-sc": "^6.0.0-alpha.18",
+    "axios": "^1.6.8",
     "bootstrap": "^5.3.3",
```

Comment:
As I mentioned we don't need bootstrap package as it has its own design system and all components in it, we have a replacement in MUI.
Also using multiple design systems will make it difficult to make our design consistent and additional features such as dark/light mode.